Tests/ForageSDKTests/ForagePINTextFieldTests.swift

```diff
@@ -119,6 +119,34 @@ final class ForagePINTextFieldTests: XCTestCase {
         XCTAssertEqual(foragePinTextField.textColor, .black)
     }
     
+    func test_backgroundColor() {
+        // Test VgsPinTextField background color
+        let foragePinTextField = ForagePINTextField()
```

Comment:
Can you explicitly test both the VGS and BT underlying wrapper classes? Basically adding in tests for the `VGSTextFieldWrapper` below just like the `BasisTheoryTextFieldWrapper`